draft-vvv-webtransport-overview.md

```diff
@@ -189,6 +197,11 @@ sessions to the network endpoints that are not WebTransport servers.
 Any transport protocol used MUST provide a way for the server to filter the
 clients that can access it by the origin {{!RFC6454}}.
 
+Any transprot protocol used MUST provide a way for a server endpoint location to
+be described using a URI {{!RFC3986}}.  They can be used for integration with
+various Web platform features that represent resources as URIs, such as Content
+Security Policy [CSP].
```

Comment:
I think you want `{{CSP}}` to make sure the links work in all output formats

Reply:
I'm not entirely sure what the difference here is, but it seems to work either way.

Reply:
Might as well be consistent and use `{{`